### PR TITLE
TTT: Corrected some strings in german translation

### PR DESCRIPTION
**Pull request by markusmarkusz**
Just corrected some things.
Like grammatic mistakes or spelling mistakes.
And i changed some translations so they sound better and are better readable.

### CODE REVIEW

**Comment by MinIsMin** [comment]
Looks good.
Please check if "L.tele_desc" has a correct inGame linebreak or some words are cut.
I can't check it right now.

(Offtopic: What is "l.search_words" for? Last words in Chat?)

**Comment by markusmarkusz** [comment]
L.search_words is last words in corpse.

File: garrysmod/gamemodes/terrortown/gamemode/lang/german.lua [hunk 1]
```diff
@@ -12,30 +12,30 @@ L.terrorists = "Terroristen"
...
 L.limit_time       = "Zeitlimit erreicht. {mapname} wird bald geladen."
-L.limit_left       = "{num} Runde(n) oder {time} Minuten verbleibend bis die Map zu {mapname} gewechselt wird."
+L.limit_left       = "{num} Runde(n) oder {time} Minute(n) verbleibend bis die Map zu {mapname} gewechselt wird."
 
 --- Credit awards
 L.credit_det_all   = "Detektive, euch wurde(n) {num} Ausrüstungs-Credit(s) für eure Performance gegeben."
 L.credit_tr_all    = "Verräter, euch wurde(n) {num} Ausrüstungs-Credit(s) für eure Performance gegeben."
 
-L.credit_kill      = "Dir wurde(n) {num} Credit(s) für das Töten eines {role} gegeben."
+L.credit_kill      = "Dir wurde(n) {num} Credit(s), da du einen {role} getötet hast."
```
Comment (Fear837) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: Shouldn't that be: Dir wurde(n) {num} Credit(s) **gegeben**, da du einen {role} getötet hast. ?

Comment (markusmarkusz) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: Oh! I've overseen this.
@svdm
Can you change it?

File: garrysmod/gamemodes/terrortown/gamemode/lang/german.lua [hunk 2]
```diff
@@ -301,19 +300,19 @@ L.item_radar      = "Radar"
...
 Person bist, die als letztes vom Opfer gesehen wurde.
 
-Schalte es im Tarnungs Reiter ein oder aus
+Schalte es im Reiter 'Tarnung' ein oder aus
 oder drücke Enter auf dem Numpad.]]
 
 -- C4
 L.c4_hint         = "Drücke {usekey} zum scharf stellen oder entschärfen."
-L.c4_no_disarm    = "Du kannst nicht das C4 eines anderen Verräters entfernen, bis dieser tot ist."
+L.c4_no_disarm    = "Du kannst das C4 eines anderen Verräters nicht entfernen, bevor dieser nicht tod ist."
```
Comment (Ayuto) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: The word ``tot`` was correct.

Comment (markusmarkusz) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: Either one of the Contributors change that or someone check if I made some more mistakes.
As a native german I can speak a perfect german but this doesn't exclude small mistakes. :/

Comment (Ayuto) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: I had a quick glance on it and the rest looks good. :)

Comment (markusmarkusz) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: @svdm, can you or @robotboy655 change that?
I hope that I didn't make any other mistakes.

File: garrysmod/gamemodes/terrortown/gamemode/lang/german.lua [hunk 3]
```diff
@@ -475,7 +474,7 @@ L.dna_desc        = [[
 Sammelt DNA-Spuren von Objekten auf
 und wird genutzt, um den Besitzer dieser ausfindig zu machen.
 
-An frischen Leichen verwenden um die DNA
+Kann an frischen Leichen verwendent werden, um die DNA
```
Comment (doctorluk) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: There is an `n` too much in _verwendent_, it should be _verwendet_

Comment (svdm) [inline garrysmod/gamemodes/terrortown/gamemode/lang/german.lua]: Thanks, fixed in ed9fa133724

**Pull request merged**